rfc/updates.md

```diff
@@ -267,3 +267,28 @@ Upgrading Constellation microservices to 2.2.0:
 Warning: Please backup any important components before upgrading Kubernetes
 Apply change [yes/No]?
 ```
+
+# Compatibility
+`constellation-upgrade` has to handle the version of four components: CLI, image, microservices and Kubernetes.
+To do this correctly and keep the cluster in a working condition some constraints are required.
+
+- Microservice, image and CLI are versioned in lockstep, i.e. each release a new version of all componentes is released.
+  Each of these version numbers is one Constellation version.
```

Comment:
I think this is somewhat hard to understand. Maybe more clear: 
`The Constellation version references all of these components`.
Still, if we want to say that each component has the same semver attached to it, we already do this in the sentence before.